README.md

```diff
@@ -72,6 +72,8 @@ It will accept a regular expression as determined by [egrep](http://linuxcommand
 
 * `only_when_mergeable`: *Optional (default: false).* If enabled only pull requests which are mergeable (all tasks done, required number of approvers reached, ...) will be built.
 
+* `max_pr_to_check`: *Optional (default: 999).* Max number of PRs to check, ordered in descending order by PR number.
```

Comment:
Our corporate bitBucket server is pretty slow, and heavily loaded. And it has days *REALLY, REALLY* slow, so after much hesitation I added this flag. For most people, nothing needs to be done, and a default of `999` is plenty enough. For the rest of us, giving it a limit speeds things up quite a bit.